evalml/tests/data_checks_tests/test_no_variance_data_check.py

```diff
@@ -29,7 +29,7 @@
 all_null_y = pd.Series([None] * 4)
 two_distinct_with_nulls_y = pd.Series(([1] * 2) + ([None] * 2))
 two_distinct_with_nulls_y_ww = two_distinct_with_nulls_y.copy()
-two_distinct_with_nulls_y_ww.ww.init()
+two_distinct_with_nulls_y_ww = ww.init_series(two_distinct_with_nulls_y_ww)
```

Comment:
What's the difference between these two functions? Why use one vs the other?

Reply:
This is because `series.ww.init()` doesn’t actually transform the underlying logical type as `WoodworkColumnAccessor` is being called, not `WoodworkTableAccessor` since this is a `series` and not a `Dataframe`.

Per [here](https://woodwork.alteryx.com/en/latest/start.html?highlight=init_series#Using-Woodwork-with-a-Series), “In cases where the LogicalType requires the Series dtype to change, a helper function ww.init_series must be used. This function will return a new Series object with Woodwork initialized and the dtype of the series changed to match the physical type of the LogicalType.”

Since the series is originally `float` due to the nans present, it needs to have its physical type properly inferred and changed to match the `IntegerNullable` dtype of `Int64`.